types/office-js-preview/index.d.ts

```diff
@@ -5104,6 +5104,25 @@ declare namespace Office {
          */
         type: EventType;
     }
+    /**
+     * Provides information about the message from the parent page that raised the DialogParentMessageReceived event.
```

Comment:
Style suggestions:
```suggestion
     * Provides information about the message from the parent page that raised the "DialogParentMessageReceived" event.
```

Reply:
If this is the code name of the event, I would recommend backticks.